.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,15 @@
+{
+    "configurations": [
+        {
+            "name": "Linux",
+            "includePath": [
+                "${workspaceFolder}/**"
+            ],
+            "defines": [],
+            "compilerPath": "/usr/bin/gcc",
+            "cppStandard": "c++11",
+            "intelliSenseMode": "clang-x64"
+        }
+    ],
+    "version": 4
+}
```

Comment:
new line

Reply:
I tried, but vscode formats this automatically and removes the extra line at the end.
This is vscode specific file, so, might be ok since it won't go thru compiler